plugins/profefe.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.8.0
+  homepage: https://github.com/profefe/kubectl-profefe
+  shortDescription: continuous profiling made easy in Kubernetes with profefe
```

Comment:
- capitalize
- no need for `made easy` (== doesn't explain what the plugin does)
- "profefe" is already plugin's name, so doesn't make a lot of sense here.